docs/reference/snapshot-restore/apis/get-snapshot-api.asciidoc

```diff
@@ -98,6 +98,38 @@ comprising the number of shards in the index, the total size of the index in
 bytes, and the maximum number of segments per shard in the index. Defaults to
 `false`, meaning that this information is omitted.
 
+`sort`::
+(Optional, string)
+Allows setting a sort order for the result. Defaults to `start_time`, i.e. sorting by snapshot start time stamp.
```

Comment:
Does this sort them ascending or descending? I'd expect ascending from these docs but in fact if using `start_time` (and probably `duration`) descending would be better. Would it make sense to have a facility to reverse the sort order too?

Reply:
> Would it make sense to have a facility to reverse the sort order too?

++ I'd vote yes here, seems useful for the UI in any case. Any objections to adding that @henningandersen ? :)

Reply:
Agree to adding this.

Reading this documentation line again, I wonder if we should leave the default sorting as "unspecified"? Would we not otherwise risk provoking an unnecessary read of snapshot info for repos that have not been upgraded when using verbose=false?

Reply:
I figured this was covered by the fact that the parameter can only be used with verbose listing. Maybe just explicitly call out that verbose=false sort order is unspecified?